packages/pages/src/dev/server/server.ts

```diff
@@ -40,6 +43,27 @@ export const createServer = async (
   // initialize the default project structure and use to help configure the dev server
   const projectStructure = await ProjectStructure.init({ scope });
 
+  if (widget) {
+    const postCssPath = await getPostCssPathFromModuleName(
```

Comment:
We're searching for the module stuff twice, once here and again in serverRenderModule. Make a function that we call first that basically determines the module path and returns an object containing all module info we'd need. That's be the path to the module folder and the module name.

We can use the above to get the postCSS file as well as load the module in serverRenderModule. The function above can do the validation if more than 1 is found.